nhais/common/data/patient.json

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
these json files should probably be in the tests/ folder that use them - they're test not application data

Reply:
Are they actually duplicates of the files already in tests/?

Reply:
I originally moved these to common since other tests needed the same json files for tests. Was going to delete these duplicates in requests -> tests -> data